corehq/apps/registration/utils.py

```diff
@@ -313,3 +321,89 @@ def send_mobile_experience_reminder(recipient, full_name, additional_email_conte
         logging.warning(
             "Can't send email, but the message was:\n%s" % message_plaintext)
         raise
+
+
+@run_only_when(not settings.ENTERPRISE_MODE and not settings.UNIT_TESTING)
```

Comment:
Should we enforce this for non-enterprise environments as well?

Reply:
https://github.com/dimagi/commcare-hq/pull/35688/commits/b48eb3154e26d569573d39cdd49cf3beb870de96